app/javascript/components/ansible-credentials-form/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import MiqFormRenderer from '@@ddf';
+import createSchema from './ansible-credential-form.schema';
+
+const AnsibleCredentialsForm = () => {
+  const [fields, setFields] = useState([]);
+  return <MiqFormRenderer 
+  schema={createSchema(fields, setFields)}
+  onSubmit={console.log}
+  onCancel={console.log}
+  />;
+};
+AnsibleCredentialsForm.propTypes = {};
+AnsibleCredentialsForm.defaultProps = {};
+export default AnsibleCredentialsForm;
```

Comment:
You're missing a newline at the end of the file, your editor should complain about that too. Also add an empty line before the export, to logically separate it from the rest.